src/docs/pages/introduction/getting-started/react.md

```diff
@@ -0,0 +1,179 @@
+- [React & CRA](https://codesandbox.io/p/github/emdgroup-liquid/liquid-sandbox-cra-tailwind/main?file=%2Fpackage.json)<br />The Sandbox uses Create React App which does not allow to adjust the webpack config. In this case we added a postinstall script to copy the assets to the public folder.
+- [React & Next.js](https://stackblitz.com/github/emdgroup-liquid/liquid-sandbox-next-tailwind?file=next.config.js)<br />Next.js uses webpack under the hood. The sandbox shows how to add a custom webpack config `next.config.js` to copy the assets to the public folder.
+
+## Usage
+
+For use in a react project, make sure to take the react bindings of Liquid Oxygen. All components are imported from `@emdgroup-liquid/liquid/dist/react`. The bindings significantly improve JSX compatibility and your developer experience.
+
+Let's have a look at how to add a [LdButton](https://liquid.merck.design/liquid/components/ld-button/) to your project. This examnple also includes a [LdIcon](https://liquid.merck.design/liquid/components/ld-icon/) as it helps you to check if Liquid Oxygen assets are loaded correctly.
+
+```tsx
+import { LdButton, LdIcon } from '@emdgroup-liquid/liquid/dist/react'
+
+export function SampleComponent() {
+  return (
+    <LdButton>
+      Click me!
+      <LdIcon name="energy" />
+    </LdButton>
+  )
+}
+```
+
+When you put this component on a page, you should see a blue button with the text "Click me!" and a lightning bolt icon.
+
+### Events
+
+Liquid Oxygen components aim to stick to the native browser components where possible. In most cases, you can expect the same events and behavior of the Liquid Oxygen component and its native equivalent. Custom events are documented on the respective component pages.
+
+Let's take our button from above and add a click handler.
+
+```tsx
+import { LdButton, LdIcon } from '@emdgroup-liquid/liquid/dist/react'
+
+export function SampleComponent() {
+  return (
+    <LdButton onClick={() => alert('Clicked!')}>
+      Click me!
+      <LdIcon name="energy" />
+    </LdButton>
+  )
+}
+```
+
+Please notice the camel case notation of the `onClick` prop. This is the expected [React convention](https://reactjs.org/docs/handling-events.html) but differs from the native `onclick` attribute and [documentation](https://www.w3schools.com/tags/ref_eventattributes.asp). As we do not explicitly document these events, you need to apply this convention yourself. Typescript and IntelliSense will aid you.
+
+There are a few cases where native events of web components do not behave as expected by React. In these cases, Liquid Oxygen provides custom events prefixed with `ld` and documented on the respective component pages.
+
+> LdInput `onChange` event invokes when the component loses focus (if the value changed), which is the standard browser behavior but differs from the native React `onChange` event. Use the `onInput` event in cases you want to handle user input immediately while typing.
```

Comment:
Maybe we should link to the [event handling guide](https://liquid.merck.design/liquid/guides/event-handling/) here.

Please use the `ld-notice` component instead of the citation markup.

Reply:
Will add a link to the dedicated guide but will keep the notice in here as react devs will probably ignore the link until they run into this issue and get frustrated. As this is basically a pretty common case I'll keep the short version of it in here. + a link to the longer event handler details